### PR TITLE
Add cron create table migrations

### PR DESCRIPTION
**Pull request by sagarnasit**
Depends on https://github.com/EasyEngine/easyengine/issues/1203

### CODE REVIEW

File: migrations/02_cron-command_create_table_cron_migration.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<?php
+namespace EE\Migration;
```
Comment (kirtangajjar) [inline migrations/02_cron-command_create_table_cron_migration.php]: Rename file as specified in - https://github.com/EasyEngine/auth-command/pull/9#discussion_r215918902

File: migrations/02_cron-command_create_table_cron_migration.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+		try {
+			self::$pdo = new \PDO( 'sqlite:' . DB );
+			self::$pdo->setAttribute( \PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION );
+		} catch ( \PDOException $exception ) {
+			EE::error( $exception->getMessage() );
+		}
+
+	}
+
+	public function up() {
```
Comment (kirtangajjar) [inline migrations/02_cron-command_create_table_cron_migration.php]: Add doc comment. Also do it in other PRs.

File: migrations/02_cron-command_create_table_cron_migration.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+		);';
+
+		try {
+			self::$pdo->exec( $query );
+		} catch ( PDOException $exception ) {
+			EE::error( 'Encountered Error while creating table: ' . $exception->getMessage() );
+		}
+	}
+
+	public function down() {
```
Comment (kirtangajjar) [inline migrations/02_cron-command_create_table_cron_migration.php]: Add doc comment.

File: README.md [hunk 4]
```diff
@@ -1,100 +1,132 @@
-# EasyEngine/cron-command
+easyengine/cron-command
```
Comment (mrrobot47) [inline README.md]: Remove `README` from this PR

File: migrations/20180910125050_cron-command_create_table_cron_migration.php [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+	/**
+	 * Execute create table query for cron table.
+	 *
+	 * @throws EE\ExitException
+	 */
+	public function up() {
+
+		$query = 'CREATE TABLE cron (
+			id INTEGER,
+			site_url VARCHAR,
```
Comment (kirtangajjar) [inline migrations/20180910125050_cron-command_create_table_cron_migration.php]: make site_url not null

File: migrations/20180910125050_cron-command_create_table_cron_migration.php [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+			command VARCHAR,
+			schedule VARCHAR,
```
Comment (kirtangajjar) [inline migrations/20180910125050_cron-command_create_table_cron_migration.php]: add user column (which could be null).

File: migrations/20180910125050_cron-command_create_table_cron_migration.php [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+	 * Execute create table query for cron table.
+	 *
+	 * @throws EE\ExitException
+	 */
+	public function up() {
+
+		$query = 'CREATE TABLE cron (
+			id INTEGER,
+			site_url VARCHAR,
+			command VARCHAR,
```
Comment (kirtangajjar) [inline migrations/20180910125050_cron-command_create_table_cron_migration.php]: make command not null

File: migrations/20180910125050_cron-command_create_table_cron_migration.php [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+			schedule VARCHAR,
+			PRIMARY KEY (id),
+			FOREIGN KEY (site_url) REFERENCES sites(site_url)
```
Comment (kirtangajjar) [inline migrations/20180910125050_cron-command_create_table_cron_migration.php]: Remove foreign key

File: migrations/20180910125050_cron-command_create_table_cron_migration.php (hunk 6 above)
Comment (kirtangajjar) [inline migrations/20180910125050_cron-command_create_table_cron_migration.php]: Make schedule not null

**Comment by sagarnasit** [comment]
Remove blocked status since it's dependency PR is merged. @kirtangajjar I have added changes you mentioned, Please check.

**Pull request merged**